### PR TITLE
Issue #64 Resolved

### PR DESCRIPTION
**Pull request by KrishnaKanth1729**
In line 75 there is a change that orders the submissions in the decreasing order of the no of votes

### CODE REVIEW

File: TWT/apps/timathon/views/judge/judge.py [hunk 1]
```diff
@@ -71,6 +72,7 @@ def get(self, request: WSGIRequest):
                 average=Avg("c1") + Avg("c2") + Avg("c3") + Avg("c4")
             )["average"]
 
-        context["submissions"] = submissions
+        context["submissions"] = sorted(
+            submissions, key=lambda x: len(x.vote_set.all()))[::-1]
```
Comment (Shubhaankar-Sharma) [inline TWT/apps/timathon/views/judge/judge.py:76]: this will make it even slower

Comment (Shubhaankar-Sharma) [reply TWT/apps/timathon/views/judge/judge.py:76]: try to sort it on line https://github.com/Tech-With-Tim/twtcodejam.net/pull/68/files#R44

**Review by Shubhaankar-Sharma — CHANGES_REQUESTED** [review]

File: TWT/apps/timathon/views/judge/judge.py (hunk 1 above)
Comment (KrishnaKanth1729) [reply TWT/apps/timathon/views/judge/judge.py]: ok done. commit fa46c6f changed that in line 44 as you requested :mhm:

**Comment by Shubhaankar-Sharma** [comment]
Needs testing, will do and merge, thanks

**Comment by KrishnaKanth1729** [comment]
@Shubhaankar-Sharma. Notify me when it is merged. Happy to contribute. Have a great day

File: TWT/apps/timathon/views/judge/judge.py [hunk 2]
```diff
@@ -33,14 +33,16 @@ def get(self, request: WSGIRequest):
             return redirect("timathon:Home")
 
         try:
-            challenge = Challenge.objects.get(ended=False, posted=True, type="MO")
+            challenge = Challenge.objects.get(
```
Comment (takos22) [inline TWT/apps/timathon/views/judge/judge.py:36]: Please use black formatting

File: TWT/apps/timathon/views/judge/judge.py [hunk 3]
```diff
@@ -33,14 +33,16 @@ def get(self, request: WSGIRequest):
...
+                ended=False, posted=True, type="MO")
         except Challenge.DoesNotExist:
             messages.add_message(
                 request, messages.INFO, "There is no ongoing challenge right now."
             )
             return redirect("/")
 
-        submissions = Submission.objects.filter(challenge=challenge)
+        submissions = Submission.objects.filter(
```
Comment (takos22) [inline TWT/apps/timathon/views/judge/judge.py:44]: Please use black formatting

**Review by takos22 — COMMENTED** [review]
LGTM, except the formatting that needs to be done on these lines with black for consistent formatting across projects

**Comment by KrishnaKanth1729** [comment]
@Shubhaankar-Sharma i went back to the previous method coz this one won't work.

File: TWT/apps/timathon/views/judge/judge.py (hunk 3 above)
Comment (KrishnaKanth1729) [reply TWT/apps/timathon/views/judge/judge.py:44]: ok sure

**Comment by Shubhaankar-Sharma** [comment]
check #69 you are the co author so dw

**Pull request closed without merging**